clayui.com/gatsby-node.js

```diff
@@ -3,6 +3,12 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
+const express = require('express');
+
 exports.createPages = require('./gatsby/createPages');
 exports.onCreateNode = require('./gatsby/onCreateNode');
 exports.onCreateWebpackConfig = require('./gatsby/onCreateWebpackConfig');
+
+exports.onCreateDevServer = ({app}) => {
+	app.use(express.static('static'));
+};
```

Comment:
Hmm this is weird that we have to do this to serve a static file, I thought this was OOTB.

Reply:
@matuzalemsteles this is only in development mode for html files. I found an old issue on the gatsby repo where it errors when using yarn develop. This was the workaround others used. I don't think they fixed it.

Reply:
No problem, I'll check this later if we still need it, the our Gatsby version is already getting out of date and they will release the v4 version, and we're still on v2.24 😅